src/ncp/spinel.h

```diff
@@ -200,6 +200,14 @@ typedef enum
     SPINEL_NET_ROLE_LEADER              = 3,
 } spinel_net_role_t;
 
+typedef enum
+{
+    SPINEL_IPV6_ICMP_PING_OFFLOAD_DISABLED       = 0,
+    SPINEL_IPV6_ICMP_PING_OFFLOAD_UNICAST_ONLY   = 1,
+    SPINEL_IPV6_ICMP_PING_OFFLOAD_MULTICAST_ONLY = 2,
+    SPINEL_IPV6_ICMP_PING_OFFLOAD_ALL            = 3,
+} spinel_ipv6_icmp_ping_offload_mode_t;
```

Comment:
With this you are essentially redefining the definition found in icmp6.h  for typedef enum otIcmp6EchoMode.  It is now required that these two definitions remain the same and that their values correspond.